cli/src/helpers.rs

```diff
@@ -483,6 +505,74 @@ pub fn command_show_solido(
     })
 }
 
+#[derive(Serialize)]
+pub struct ShowSolidoAuthorities {
+    #[serde(serialize_with = "serialize_b58")]
+    pub solido_program_id: Pubkey,
+
+    #[serde(serialize_with = "serialize_b58")]
+    pub solido_address: Pubkey,
+
+    #[serde(serialize_with = "serialize_b58")]
+    pub reserve_account: Pubkey,
+
+    #[serde(serialize_with = "serialize_b58")]
+    pub stake_authority: Pubkey,
+
+    #[serde(serialize_with = "serialize_b58")]
+    pub mint_authority: Pubkey,
+
+    #[serde(serialize_with = "serialize_b58")]
+    pub rewards_withdraw_authority: Pubkey,
+}
+
+impl fmt::Display for ShowSolidoAuthorities {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
The program id and reserve account are part of the struct, but not printed here. Is that intentional?

Reply:
The reserve account is printed, I think is ok that the program id is not printed, since the user has to provide it to call the cmd.